### PR TITLE
Update docs for course folders feature in 2020.2.0+

### PR DESCRIPTION
**Pull request by cbeach47**
Just realized I should have made these changes on my fork.  Since this is fairly minor, I'll leave this PR as is.

### CODE REVIEW

File: guides/BlackboardLearnIntegrationGuide.md [hunk 1]
```diff
@@ -107,14 +107,26 @@ _Note: openEQUELLA prepends a unique value (by **LTI Consumer**) to the user ID.
 7. Under **Tool Provider Information**, set the **Tool Provider URL** to `{https://your.oE.domain.xyz/demo/}ltisignon.do`.
 8. The **Tool Provider Key** and **Tool Provider Secret** will be preconfigured (and readonly).
 
+#### Alternative configuration
+New in oEQ 2020.2.0+, the LTI integration now supports leveraging the course folder structure in the selection session.
+
+To enable, use the steps above with the following changes.  Note this will require you to configure a REST integration flow (as described in this document) as well.
+
+1. For the URL, add the following parameters:  `{https://your.oE.domain.xyz/demo/}ltisignon.do??action=structured&connector_uuid={backing-oeq-external-system-connector}
```
Comment (ChristianMurphy) [inline guides/BlackboardLearnIntegrationGuide.md]: not for necassary this PR,
but sometime we should settle on a consistent {your domain goes here} link in the docs, that can be added to the allow list for link checking to make the valid/broken link checker happy.

**Review by ChristianMurphy — APPROVED** [review]

File: guides/BlackboardLearnIntegrationGuide.md (hunk 1 above)
Comment (cbeach47) [reply guides/BlackboardLearnIntegrationGuide.md]: I added that to the agenda for the next CDM

Comment (edalex-ian) [inline guides/BlackboardLearnIntegrationGuide.md]: Looks like that has a stray `?` and missing ending back tick:

```suggestion
1. For the URL, add the following parameters:  `{https://your.oE.domain.xyz/demo/}ltisignon.do?action=structured&connector_uuid={backing-oeq-external-system-connector}`
```

**Review by edalex-ian — APPROVED** [review]
Just one minor typo(s). But GTG.

**Pull request merged**